### DISCUSSION TITLE
"Kibana server is not ready yet" and Kibana index migration

### DISCUSSION THREAD

**Discussion by dougburks** [Security-Onion-Solutions/securityonion#3639] [Announcements]
Several folks have experienced issues after upgrading with Kibana displaying `Kibana server is not ready yet` even after waiting a few minutes for it to fully initialize. In most cases, this is a fairly simple issue with Kibana index migration and is quickly resolved by following the steps at https://docs.securityonion.net/en/2.3/kibana.html#diagnostic-logging.

This is a common issue when upgrading the Elastic Stack (not just with Security Onion). It looks like Elastic has fixed this in Kibana 7.12, so hopefully this won't be an issue for future upgrades.

**Comment by cerberus10748** [comment]
Worked like a charm for me, thanks for the link to the fix Doug !

**Reply by corrcorr** [reply]
HI

I have the same error but Kibana logs is showing this error

{"type":"log","@timestamp":"2021-06-14T11:15:10+00:00","tags":["info","plugins","monitoring","monitoring"],"pid":294,"message":"config sourced from: production cluster"}
{"type":"log","@timestamp":"2021-06-14T11:15:11+00:00","tags":["info","savedobjects-service"],"pid":294,"message":"Waiting until all Elasticsearch nodes are compatible with Kibana before starting saved objects migrations..."}
{"type":"log","@timestamp":"2021-06-14T11:15:12+00:00","tags":["info","savedobjects-service"],"pid":294,"message":"Starting saved objects migrations"}
{"type":"log","@timestamp":"2021-06-14T11:15:12+00:00","tags":["warning","savedobjects-service"],"pid":294,"message":"Unable to connect to Elasticsearch. Error: search_phase_execution_exception"}
{"type":"log","@timestamp":"2021-06-14T11:15:12+00:00","tags":["warning","savedobjects-service"],"pid":294,"message":"Unable to connect to Elasticsearch. Error: search_phase_execution_exception"}

**Reply by dougburks** [reply]
@corrcorr Since your Kibana logs do not indicate an issue with Kibana index migration, please create a new Discussion instead of replying to this one.  Thanks!
